code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1348,21 +1353,32 @@ var/list/rank_prefix = list(\
 	reset_view(A)
 
 /mob/living/carbon/human/proc/resuscitate()
-	var/obj/item/organ/internal/vital/heart_organ = random_organ_by_process(OP_HEART)
-	var/obj/item/organ/internal/vital/brain_organ = random_organ_by_process(BP_BRAIN)
+	
+	var/obj/item/organ/internal/vital/heart/heart_organ = random_organ_by_process(OP_HEART)
+	var/obj/item/organ/internal/vital/brain/brain_organ = random_organ_by_process(BP_BRAIN)
+
+	if((!heart_organ || heart_organ.is_broken()) && (!brain_organ) || brain_organ.is_broken())
+		visible_message(SPAN_WARNING("\The [src] lay still, devoid of any hint of vitality or warmth."))
+		return 0
+
+	if(!heart_organ || heart_organ.is_broken())
+		visible_message(SPAN_WARNING("\The [src] twitches a bit, but their body remains lifeless, unresponsive to any stimulus."))
+		return 0
 
-	if(!is_asystole() && !(heart_organ && brain_organ) || (heart_organ.is_broken() || brain_organ.is_broken()))
+	if(!brain_organ || brain_organ.is_broken())
+		visible_message(SPAN_WARNING("\The [src] changes is colour, but remains tranquil and utterly still."))
 		return 0
 
 	if(world.time >= (timeofdeath + NECROZTIME))
+		visible_message(SPAN_WARNING("\The [src] shows no signs of reaction, a grim reminder of the boundaries that separate life and death."))
 		return 0
 
 	var/oxyLoss = getOxyLoss()
 	if(oxyLoss > 20)
 		setOxyLoss(20)
 
 	if(health <= (HEALTH_THRESHOLD_DEAD - oxyLoss))
-		visible_message(SPAN_WARNING("\The [src] twitches a bit, but their body is too damaged to sustain life!"))
+		visible_message(SPAN_WARNING("\The [src] twitches a bit, but their body is too damaged to sustain life."))
```

Comment:
I think the message could be a bit more clear. It's better than nothing but without looking at the code, I'm not sure people will fully understand what the messages indicate in term of missing heart/brain.

Reply:
> I think the message could be a bit more clear. It's better than nothing but without looking at the code, I'm not sure people will fully understand what the messages indicate in term of missing heart/brain.

I'll redo message part to be more depended on stats of people looking